README.md

```diff
@@ -2,37 +2,38 @@
 
 ## Introduction
 
-A module that tracks views and downloads of Islandora items, using the Drupal database.
+A module that tracks views and downloads of Islandora items, as well as solr searches, in the Drupal database.
 
 Features include:
 
-* Toggle to ignore common bots, with a configurable regex bot filter
-* View count uses session variables, and defaults to a 5 minute cooldown for repeated requests
-* IP Exclusion list to prevent artificially inflating counts while testing/developing/administrating
-* Logs all views and datastream downloads
+* ability to ignore common bots, with a configurable regex bot filter
+* ability to ignore repeated requests based on session variables, with a default cooldown period of 5 minutes
+* ability to ignore requests from specific IP addresses for testing/developing/administration
 * Views integration
 
-Note:
+Islandora Usage Stats provides the back-end framework and gathers usage data, which can be exposed through custom Views or additional modules. It also provides some built-in tools for exposing usage information.
+
+Caution:
 
 * This module, and the views/blocks it generates, does **not** respect XACML or namespace restrictions.
-* As this is a server-side tracking solution, a caching layer could prevent accesses from being recorded.  If this is impacting you a [solution](https://github.com/discoverygarden/islandora_ga_reports) using JavaScript may work better.
+* As this is a server-side tracking solution, a caching layer can prevent usage from being accurately recorded.  If this is impacting you, a solution using JavaScript may work better for you. Community solutions include [discoverygarden's Google Analytics module](https://github.com/discoverygarden/islandora_ga_reports) and [Diego Pino's Matomo module](https://github.com/DiegoPino/islandora_piwik/). 
 
 ## Requirements
 
 This module requires the following modules/libraries:
 
-* [Tuque](https://github.com/islandora/tuque)
 * [Islandora](https://github.com/islandora/islandora)
 * [Islandora basic collection](https://github.com/Islandora/islandora_solution_pack_collection)
-* [Views (3.x)](https://www.drupal.org/project/views)
+* [Views](https://www.drupal.org/project/views)
+* [Views Data Export](https://www.drupal.org/project/views_data_export)
 * [Date](https://www.drupal.org/project/date)
 * [Datepicker](https://www.drupal.org/project/datepicker)
-* [Views Data Export](https://www.drupal.org/project/views_data_export)
 
 This module can be extended with:
```

Comment:
Maybe expand this further to make it even more explicit. "The following contributed modules" or "the following community/Drupal modules"